calc/src/mechanics/gen8.ts

```diff
@@ -1053,6 +1050,10 @@ function calculateGen8(
     finalMods.push(0x800);
     description.defenderAbility = defender.ability;
   }
+  if (move.isSound && attacker.hasAbility('Punk Rock')) {
+    finalMods.push(0x1300);
```

Comment:
What’s wrong with 0x14cd?

Reply:
Also what do you mean by “checked twice”?

Reply:
You're right, my bad, that was probably some leftover from testing. Changed it back to 0x14cd.

Reply:
Sweet, just check the field for this check then :)